pkg/ui/src/views/jobs/duration.tsx

```diff
@@ -30,8 +29,8 @@ export class Duration extends React.PureComponent<{ job: Job }> {
     const started = TimestampToMoment(this.props.job.started);
     const finished = TimestampToMoment(this.props.job.finished);
     const modified = TimestampToMoment(this.props.job.modified);
-    if (jobHasOneOfStatuses(this.props.job, JOB_STATUS_PENDING, JOB_STATUS_PAUSED)) {
-      return _.capitalize(this.props.job.status);
+    if (jobHasOneOfStatuses(this.props.job, JOB_STATUS_PENDING)) {
+      return "Waiting for GG TCL";
```

Comment:
The message in the design was there as an example not as a permanent status for all pending jobs. The actual message in the code is `Waiting for GC TTL` and is set on the job as its "running_status" field.

I think for now, if the job has a running status, we should just show it underneath the badge or progress bar. I will confirm with Anne on this as well.